clients/unshaded/src/main/java/tachyon/client/FileOutStream.java

```diff
@@ -123,7 +123,8 @@ public void close() throws IOException {
       } catch (IOException ioe) {
         if (mWriteType.isMustCache()) {
           LOG.error(ioe.getMessage(), ioe);
-          throw new IOException("Fail to cache: " + mWriteType, ioe);
+          throw new IOException("Fail to cache: " + mWriteType + ", message: " + ioe.getMessage(),
+              ioe);
```

Comment:
I think its superfluous to add the message since the entire exception will be logged?